packages/lesswrong/server/scripts/sendWrappedNotifications.ts

```diff
@@ -12,14 +11,16 @@ const sendWrappedNotifications = async () => {
   const readStatuses = await ReadStatuses.find({
     isRead: true,
     lastUpdated: {$gte: start, $lte: end},
-    postId: {$exists: true, $ne: null}
+    postId: {$ne: null}
   }, {projection: {userId: 1}}).fetch()
   
   const userIds = uniq(readStatuses.map(rs => rs.userId))
-  // filter out users who are banned
+  // filter out users who are banned, deleted, or have unsubscribed from all emails
   const users = await Users.find({
     _id: {$in: userIds},
-    banned: {$exists: false}
+    banned: {$exists: false},
+    unsubscribeFromAll: {$ne: true},
```

Comment:
I think we are not consistently blocking emailing users who unsubscribed from all, so I'm excluding them out of caution